sum.py

```diff
@@ -1,6 +0,0 @@
-sum=0;//variable to store sum
```

Comment:
Remove this file and push

git rm sum.py

git commit -m "Task-2 : Removed sum.py as a new file sum_range.py replaces the functionality of that"

git push -u origin <branch_name>